crates/containerd-shim-wasmedge/src/executor.rs

```diff
@@ -98,6 +100,27 @@ impl WasmEdgeExecutor {
 
         Ok(vm)
     }
+
+    fn can_handle(&self, spec: &Spec) -> Result<(), ExecutorValidationError> {
```

Comment:
This doens't use self. It coud be a free function.

Reply:
Same applies to prepare if we move `self.stdio.redirect()?` outside the function

Reply:
I will move the `can_handle` to free function. I will leave the `prepare` for future PR.